nodestream_plugin_neptune/database_connector.py

```diff
@@ -1,17 +1,48 @@
-from nodestream.databases import TypeRetriever, DatabaseConnector
-from nodestream.databases.query_executor import QueryExecutor
+from nodestream.databases.copy import TypeRetriever
+from nodestream.databases.database_connector import QueryExecutor, DatabaseConnector
+from .ingest_query_builder import NeptuneDBIngestQueryBuilder
+from typing import Any, Dict, List, Optional, Tuple, Union
 
-from .query_executor import NeptuneQueryExecutor
-from .type_retriever import NeptuneTypeRetriever
+class NeptuneDatabaseConnector(DatabaseConnector, alias="neptune"):
+    @classmethod
+    def from_file_data(
+        cls,
+        host: str,
+        region: str,
+        database_name: str = "neptune",
```

Comment:
I think this can be removed. If this is a carry over from the `neo4j` connector, this is the name of the logical database to insert data into. If memory serves, `neptune` is a single database per cluster/instance. At the very least it is being dropped here.

Reply:
Yup that's correct. It's a carry over from neo4j. Should be removed in latest revision.